src/connect_kubernetes_api_run_job.workflows.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START workflows_connect_kubernetes_api_run_job]
+main:
+  steps:
+    - init:
+        assign:
+          - project: "PROJECT_ID"
+          - location: "LOCATION"
+          - cluster_id: "CLUSTER_NAME"
+          - job_name: "JOB_NAME"
+          - namespace: "default"
+    - create_job:
+        call: gke.create_job
+        args:
+          cluster_id: '${cluster_id}'
+          location: '${location}'
+          project: '${project}'
+          namespace: '${namespace}'
+          job:
+            apiVersion: batch/v1
+            kind: Job
+            metadata:
+              name: "${job_name}"
+            spec:
+              template:
+                spec:
+                  containers:
+                    - name: counter
+                      image: centos:7
```

Comment:
**Thought:** (not action needed)
Normally, I would suggest specifying the minor, patch, and digest numbers/strings (e.g., replacing `centos:7` with `centos:7.9.2009@sha256:dead07b4d8ed7e29e98de0f4504d87e8880d4347859d839686a31da35a3b532f`) of every `image` that we use. This would ensure that the user pulls the _exact_ version of the image that we have tested, and any bumps to the version will be tested (by us) during pull-requests to this git repository.

However, this repository doesn't test the deployment (in its automated pull-request checks/tests), so let's let it slide.

Reply:
Ack. Thanks, Nim.